draft-ietf-rtcweb-jsep.xml

```diff
@@ -1644,7 +1644,10 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                     candidate to the ICE Agent, which, if parsed
                     successfully, will be added to the current and/or
                     pending remote description according to the rules
-                    defined for Trickle ICE. Connectivity checks will
+                    defined for Trickle ICE.
+                    If the MID or m-line index provided in the ICE candiate are
+                    invalid, an error is generated. 
```

Comment:
Should also mention the actual candidate info itself, e.g. "If the MID, m-line index, or candidate string is invalid,"